test/images/logs-generator/Dockerfile

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google_containers/ubuntu-slim:0.6
+FROM BASEIMAGE
 
+ENV LOGS_GENERATOR_LINES_TOTAL 1
+ENV LOGS_GENERATOR_DURATION 1s
 
 COPY logs-generator /
 
-CMD /logs-generator --logtostderr \
-  --log-lines-total=${LOGS_GENERATOR_LINES_TOTAL} \
-  --run-duration=${LOGS_GENERATOR_DURATION}
+CMD ["sh", "-c", "/logs-generator --logtostderr --log-lines-total=${LOGS_GENERATOR_LINES_TOTAL} --run-duration=${LOGS_GENERATOR_DURATION}"]
```

Comment:
This is because of - https://github.com/moby/moby/issues/5509#issuecomment-42173047, anyways I'll revert back to original format without rectangle bracket.